crates/red_knot_python_semantic/src/types/class.rs

```diff
@@ -332,10 +332,16 @@ impl<'db> Class<'db> {
         //     from the non-dynamic members of the class's MRO.
         let mut dynamic_type_to_intersect_with: Option<Type<'db>> = None;
 
-        let mut lookup_result: LookupResult<'db> = Err(LookupError::Unbound);
+        let mut lookup_result: LookupResult<'db> =
+            Err(LookupError::Unbound(TypeQualifiers::empty()));
 
         for superclass in self.iter_mro(db) {
             match superclass {
+                ClassBase::Dynamic(DynamicType::TodoProtocol) => {
+                    // TODO: We currently skip protocols when looking up class members, in order to
```

Comment:
Nit: we don't "skip protocols", we "skip Protocol" -- it is the base type `Protocol` itself that we skip, not "protocols" (which is all classes inheriting `Protocol`).